test-data/unit/stubgen.test

```diff
@@ -1270,6 +1274,78 @@ class Generator: ...
 
 def f() -> _Generator[Incomplete, None, None]: ...
 
+[case testGeneratorYieldFrom]
+def g1():
+    yield from x
+def g2():
+    y = yield from x
+def g3():
+    yield from x
+    return
+def g4():
+    yield from x
+    return None
+def g5():
+    yield from x
+    return z
+def g6():
+    y = yield from x
+    return z
+
+[out]
+from _typeshed import Incomplete
+from collections.abc import Generator
+
+def g1() -> Generator[Incomplete, None, None]: ...
+def g2() -> Generator[Incomplete, Incomplete, None]: ...
```

Comment:
Why is the send type different depending on whether the `yield from` value is used? The value returned from the `yield from` is what the inner generator returns, it doesn't relate to the send type.

Reply:
Thanks, forgot that `yield from` has the send type of its sub generator. Fixed in 3054b5c.